network/server.go

```diff
@@ -94,12 +97,18 @@ type Server struct {
 	joinWatchersLock sync.Mutex
 
 	emitterPeerEvent event.Emitter
+
+	inboundConnCount int64
+
+	tempeoraryDials sync.Map
```

Comment:
I think there's a typo here.

Reply:
Thanks for catch Commit [link](https://github.com/0xPolygon/polygon-sdk/pull/343/commits/62ade33b25c2f45eafb7a77384c29c18cc01a8ce)